frontend-react/e2e/pages/BasePage.ts

```diff
@@ -0,0 +1,202 @@
+        }
+    }
+
+    addRouteHandlers(items: RouteHandlerEntry[]) {
+        for (const [url, _fulfillOptions] of items) {
+            const handler: RouteHandlerFn = async (route, request) => {
+                const { isMock, ...fulfillOptions } =
+                    typeof _fulfillOptions === "function"
+                        ? _fulfillOptions(request)
+                        : _fulfillOptions;
+                const mockErrorFulfillOptions = isMock
+                    ? typeof this.mockError === "function"
+                        ? this.mockError(request)
+                        : this.mockError
+                    : undefined;
+                const mockCacheFulfillOptions = isMock
+                    ? await this.getMockCacheFulfillOptions(url)
+                    : undefined;
+                const mockOverrideFulfillOptions =
+                    mockErrorFulfillOptions ?? mockCacheFulfillOptions;
+
+                if (isMock && !this.isMocked)
+                    throw new Error("Mocks are disabled");
+
+                return await route.fulfill(
+                    mockOverrideFulfillOptions ?? fulfillOptions,
+                );
+            };
+            this.routeHandlers.set(url, handler);
+        }
+    }
+
+    addMockRouteHandlers(items: RouteHandlerEntry[]) {
+        return this.addRouteHandlers(
+            items.map(([url, _fulfillOptions]) => {
+                if (typeof _fulfillOptions === "object") {
+                    return [
+                        url,
+                        {
+                            isMock: true,
+                            ..._fulfillOptions,
+                        },
+                    ];
+                }
+
+                const fn: RouteFulfillOptionsFn = (request) => {
+                    return {
+                        isMock: true,
+                        ..._fulfillOptions(request),
+                    };
+                };
+                return [url, fn];
+            }),
+        );
+    }
+
+    addDefaultRouteHandlers() {
+        return this.addRouteHandlers([
+            // Azure Application Insights Tracking
+            [
+                "*.in.applicationinsights.azure.com/v2/track",
+                (request) => {
+                    const itemsCount = (request.postDataJSON() as any[]).length;
+                    return {
+                        json: {
+                            itemsReceived: itemsCount,
+                            itemsAccepted: itemsCount,
+                            appId: null,
+                            errors: [],
+                        },
+                    };
+                },
+            ],
+            // Azure Application Insights Config
+            [
+                "https://js.monitor.azure.com/scripts/b/ai.config.*.cfg.json",
+                {
+                    json: appInsightsConfig,
+                },
+            ],
+            // Google Analytics
```

Comment:
Can we just use the response from the `fulfillGoogleAnalytics()` method in our `utils.ts `file here?